pkg/controller/stacks/templates/stackdefinition_controller_test.go

```diff
@@ -93,8 +93,8 @@ spec:
    - "private network"
    - "cheap"
    - "minimal"
-  website: "httos://upbound.io"
-  source: "https://github.com/muvaf/minimal-gcp"
+  website: "https://upbound.io"
```

Comment:
nice catch!

Reply:
Note that I just opened PR's to change these to `https://crossplane.io` and company to `Crossplane` on the sample stacks. These values don't really man too much here, but update if you wish :)